manage.py

```diff
@@ -50,17 +52,25 @@ def test(all=False):
 
 
 @manager.command
-def scrape(noqueue=False):
+def scrape():
+    """Runs one live scrape of all feeds."""
+
+    for feed in Feed.query.all():
+        scrape_feed_articles(feed)
+        categorize_feed_articles(feed, categorizer)
+
+
+@manager.command
+def scrape_loop():
     """Start an infinte loop to scrape & categorize articles."""
 
-    if noqueue:
-        scrape_and_categorize_articles()
+    q = Queue('default', connection=conn)
+    while True:
 
-    else:
-        q = Queue('default', connection=conn)
-        while True:
-            job = q.enqueue_call(func=scrape_and_categorize_articles, timeout=1800)
-            time.sleep(THIRTY_MINUTES)
+        for feed in Feed.query.all():
+            q.enqueue_call(func=categorize_feed_articles,
+                           args=(feed, categorizer,), timeout=1800)
+        time.sleep(THIRTY_MINUTES)
```

Comment:
I think we should keep it. Because we're just enqueuing the tasks, it happens in seconds. If we get rid of the sleep, a categorize_feed_articles task will get added to the queue every second. This will lead to a big backlog of tasks that I don't think we want.